package.json

```diff
@@ -8,11 +8,14 @@
   "scripts": {
     "dev": "webpack-dev-server --progress --config build/webpack.dev.conf.js",
     "start": "NODE_ENV=development npm run dev",
-    "test": "npm run unit",
+    "test": "vue-cli-service test:unit",
     "lint": "eslint --ext .js,.vue src test/unit/specs",
     "build": "node build/build.js"
   },
   "dependencies": {
+    "@vue/cli-plugin-unit-jest": "^3.2.0",
```

Comment:
This should be devDependencies?

Reply:
That's what I thought, but for some reason the vue cli put this in dependencies (and I followed their lead).  I could probably change it, I just wanted to get this fix out quickly since the issue was stale.